router/batchrouter/handle_async.go

```diff
@@ -104,8 +105,11 @@ func (brt *Handle) pollAsyncStatus(ctx context.Context) {
 								if err != nil {
 									panic(err)
 								}
-
 								importingList := list.Jobs
+								var GetUploadStatsInput common.FetchUploadJobStatus
+								GetUploadStatsInput.FailedJobURLs = asyncResponse.FailedJobURLs
+								GetUploadStatsInput.Parameters = importingJob.LastJobStatus.Parameters
+								GetUploadStatsInput.ImportingList = importingList
```

Comment:
```suggestion
								var getUploadStatsInput common.FetchUploadJobStatus
								getUploadStatsInput.FailedJobURLs = asyncResponse.FailedJobURLs
								getUploadStatsInput.Parameters = importingJob.LastJobStatus.Parameters
								getUploadStatsInput.ImportingList = importingList
```

The variable need not be exported

Reply:
addressed [here](https://github.com/rudderlabs/rudder-server/pull/3623)